### PR TITLE
Fix darknet when scaling or cropping enabled

### PR DESCRIPTION
**Pull request by mattdawkins**

### CODE REVIEW

**Comment by mattdawkins** [comment]
Note: I cherry-picked this from the VIAME branch

File: arrows/darknet/darknet_detector.cxx [hunk 1]
```diff
@@ -275,15 +276,15 @@ check_configuration( vital::config_block_sptr config ) const
...
 vital::detected_object_set_sptr
 darknet_detector::
 detect( vital::image_container_sptr image_data ) const
 {
   kwiver::vital::scoped_cpu_timer t( "Time to Detect Objects" );
 
-  cv::Mat cv_image = kwiver::arrows::ocv::image_container::vital_to_ocv( image_data->get_image() );
-  cv::Mat cv_resized_image;
+  cv::Mat cv_resized_image, cv_image =
+    kwiver::arrows::ocv::image_container::vital_to_ocv( image_data->get_image() );
```
Comment (linus-sherrill) [inline arrows/darknet/darknet_detector.cxx]: Not my preferred way to define two variables.

Comment (mleotta) [inline arrows/darknet/darknet_detector.cxx]: I agree.  This struck me as odd.  I liked to old way better.  It was more clear.  This way is more likely to confuse a C++ novice.  One might assume that you are trying to assign the result of the function to both variables.

File: arrows/darknet/darknet_detector.cxx [hunk 2]
```diff
@@ -275,15 +276,17 @@ check_configuration( vital::config_block_sptr config ) const
...
 detect( vital::image_container_sptr image_data ) const
 {
   kwiver::vital::scoped_cpu_timer t( "Time to Detect Objects" );
 
-  cv::Mat cv_image = kwiver::arrows::ocv::image_container::vital_to_ocv( image_data->get_image() );
-  cv::Mat cv_resized_image;
+  cv::Mat cv_image =
+    kwiver::arrows::ocv::image_container::vital_to_ocv( image_data->get_image() );
+
+  cv_resized_image;
```
Comment (mleotta) [inline arrows/darknet/darknet_detector.cxx]: isn't this missing the type (cv::Mat) now?

Comment (mattdawkins) [inline arrows/darknet/darknet_detector.cxx]: Details, pff

**Pull request merged**